code/game/objects/items/weapons/null_rod.dm

```diff
@@ -173,6 +173,29 @@
 	item_state = "katana"
 	fluff_pickup = "bisect"
 
+/obj/item/weapon/nullrod/crucifix
+	name = "crucifix"
+	desc = "A shiny gold crucifix."
+	icon_state = "crucifix"
+	item_state = "crucifix"
+	fluff_pickup = "turn"
+
+/obj/item/weapon/nullrod/crucifix/pickup(mob/living/carbon/human/user)
+	if(user.mind.assigned_role == "Chaplain")
+		desc = "What you are about to do has not been approved by the Space Vatican."
+	else
+		desc = initial(desc)
+
+/obj/item/weapon/nullrod/crucifix/attack_self(mob/user as mob) //For larping during an exorcism
+	if(user.mind.assigned_role == "Chaplain")
```

Comment:
```suggestion
	if(user.mind?.assigned_role == "Chaplain")
```
If, for whatever reason, the user no longer has a mind, this will prevent a runtime.